### ISSUE TITLE
Minor UI issue: "Choose File" opens LBRY install folder

### ISSUE THREAD

**Issue by LavRadis** [lbryio/lbry-desktop#1811]
## The Issue
On the first usage, the "Choose File" button in the "Publish" section opens in LBRY install directory. It could be confusing for new users. It could maybe open in "My documents" or "My computer". It remembers the last directory after the first time.

## System Configuration

- LBRY Daemon version: 0.20.4
- LBRY App version: 0.23.0-rc.3
- LBRY Installation ID: 7tYN144rHXrspzJG9xLP28fi1JztXnbpKro6DBJEQyzBmXTyDgqRLfYsUvHYFVmMe2
- Operating system: Win 10

**Comment by seanyesmunt** [comment]
On mac it opens to the folder you chose from last time after clicking the button. I guess this isn't happening on windows? Must be an OS thing

**Comment by tzarebczan** [comment]
Yes, that's true on windows also. But what happens when it's on a first run (no file previously selected)?

**Comment by seanyesmunt** [comment]
Seems like it's persisting between first runs. We might need to try on a new machine?

**Comment by bcrowie** [comment]
I'd like to take this one on if that's alright.

**Comment by seanyesmunt** [comment]
@bcrowie It's all yours!

**Comment by DispatchCommit** [comment]
Set `defaultPath` for the `dialog.showOpenDialog()` function: https://www.electronjs.org/docs/api/dialog#dialogshowopendialogbrowserwindow-options

You'd also likely need to detect which platform a user is on and/or use something like `%HOMEPATH%` to set it to the user's home profile (for windows at least).
